package.json

```
@@ -44,6 +44,7 @@
         "@interfaces": "dist/interfaces"
     },
     "dependencies": {
+        "@hubspot/api-client": "^4.1.0",
```

Lets always use fixed versions. Gives us more control.
```suggestion
        "@hubspot/api-client": "4.1.0",
```